### ISSUE TITLE
[Feature req] Easier way of importing games.

### ISSUE THREAD

**Issue by Twig6943** [Heroic-Games-Launcher/HeroicGamesLauncher#4230]
### Problem description

(Talking about a fresh install where I have all my games installed to a seperate drive)

On steam I just point the install directory and then it picks up every game I have (takes less than 20 seconds)

On heroic I have to add every single game manually (also specify the prefixes)

### Feature description

Use steam's approach to fix this problem aka by introducing library folders which can be imported very quickly (Just like steam)

### Alternatives

N/A

### Additional information

This is torture to do. I have over 30 games (Most are sideloaded)

**Comment by arielj** [comment]
I think this is a duplicate of https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/3377 (it mentions creating a folder/library to point to and import all games inside it)

**Comment by incansvl** [comment]
I agree with and support the basic requirement (bulk import an existing library of games). Some additional comments-

1. For games installed via a "foreign" launcher like Steam or GoG, the ideal would be to scrape the game data from that launcher, but that depends on the foreign launcher either having an API to do this, and not deliberately making it difficult.
2. Next best is to be able to import all the games installed under a specified root folder. Using the analogy with the way music players like MediaMonkey can import new music tracks, the options (in increasing order of difficulty and desirability) would be-
3.1 A "single hit" import function that is triggered from the user interface and scans a specified folder tree.
3.2 An "add folder to libraries" option that scans the specified library folders at startup of Heroic for new games.~
3.3 A "folder watcher" feature, like the above but that will also catch new games that are added on the fly by monitoring file system changes.
4. A "nice to have" option would be to have the facility to add multiple folders under a custom category (e.g. "GoG Games") for cases where users might have their game launcher libraries split across folders and/or drives.

**Comment by Y3sMan** [comment]
It would also be nice to have a "Duplicate" button to quickly copy an entire game's config and prefix. I have tons of alternate exes I need to run for a single game, and having to open one, copy the prefix, open the other, enter the name, *then* enter the prefix (because setting the prefix *before* setting the game name will overwrite the prefix setting.
